src/_utils/icon/BaseIcon.tsx

```diff
@@ -18,6 +18,7 @@ export const BaseIconDefaultProps = {
   title: '',
   badgeAriaLabel: '',
   badgeContent: '',
+  isDisabled: false,
```

Comment:
Why not disabled?

Reply:
Not sure, it will make it consistent with html but inconnsistent with the component API (isLoading, isChecked).
I would keep it as it is

Reply:
Yes there's both `disabled` and `isDisabled` in some of the files I touched.
Personally, I prefer the explicit boolean wording `isDisabled`.

Reply:
`isDisabled` is indeed a nicer API but it is not the one used by HTML and the community, which means that we are breaking the _implicit_ API for disabling things. If we need to pass it to some external component, or someone external wants to use it, it should be wrapped to rename the prop 😟.

Reply:
I'm keeping it for now but you're right, the API should be streamlined in a separate PR.